TutorialApplication/MainViewController.m

```diff
@@ -145,4 +159,9 @@ - (void)tableView:(UITableView *)tableView commitEditingStyle:(UITableViewCellEd
     }
 }
 
+-(void) dealloc
+{
+    [[NSNotificationCenter defaultCenter] removeObserver:self];
```

Comment:
You should unsubscribe only from notifications you have registered for, not for everything.